charts/datadog/values.yaml

```diff
@@ -254,6 +254,19 @@ datadog:
   ## ref: https://docs.datadoghq.com/getting_started/tagging/assigning_tags/?tab=containerizedenvironments#environment-variables
   checksCardinality:  # low, orchestrator or high (not set by default to avoid overriding existing DD_CHECKS_TAG_CARDINALITY configurations, the default value in the Agent is low)
 
+  # SBOM configuration
+  sbom:
+    # datadog.sbom.enabled -- Enable SBOM (Software Bill Of Materials) collection
+    enabled: false
+
+    containerImage:
+      # datadog.sbom.containerImage.enabled -- Enable SBOM collection for container images
+      enabled: false
```

Comment:
I think we can enable by default `datadog.sbom.containerImage.enabled` and set it only if `datadog.sbom.enabled: true`.

For `host` we can do the same when it is supported on the backend side.